{
  FalseOn5xx function=new FalseOn5xx();
  RuntimeException exception=new RuntimeException();
  try {
    function.create(exception);
  }
 catch (  Exception ex) {
    assertEquals(ex,exception);
  }
}
